src/components/task-plan/homework.cjsx

```diff
@@ -227,6 +241,31 @@ ExerciseSummary = React.createClass
   removeTutorSelection: ->
     TaskPlanActions.updateTutorSelection(@props.planId, -1)
 
+  componentDidMount: () ->
+    el = @getDOMNode()
+    @staticPosition = @getPosition(el)
+    @handleScroll() # Update scroll position immediately on mount
+    @optimizedScrollFunc = throttle('scroll', 'optimizedScroll')
+    window.addEventListener('optimizedScroll', @handleScroll)
+
+  componentWillUnmount: () ->
+    window.removeEventListener('scroll', @optimizedScrollFunc)
+    window.removeEventListener('optimizedScroll', @handleScroll)
+
+  getPosition: (el) -> el.getBoundingClientRect().top - document.body.getBoundingClientRect().top
+
+  handleScroll: (e) ->
+    el = @getDOMNode()
+    if not el.textContent then return # HACK: Ignore empty span that is also an ExerciseSummary
+
+    if document.body.scrollTop + 60 > @staticPosition
+      el.classList.add('navbar', 'navbar-fixed-top', 'navbar-fixed-top-lower')
+      document.body.style.marginTop = '120px'
```

Comment:
The margin is there to offset the page jumping when the exercise summary becomes a fixed element, so it's very closely tied with the height and padding of that element.

So I'm not sure making it a LESS style would be entirely appropriate, although it would certainly be more resilient to calculate how much to offset the body rather than the hardcoded 120px (it was hardcoded as a bit of a shortcut since the calculation is actually quite a few lines and requires only performing the calculation when the element is not fixed).